internal/sync.go

```diff
@@ -468,6 +467,56 @@ func (s *syncGSuite) SyncGroupsUsers(query string) error {
 	return nil
 }
 
+// getAllGroupMembers return a slice of google group members of googleGroups
+// whereby all the members are of USER type and inherited groups are resolved.
+func (s *syncGSuite) getAllGroupMembers(group *admin.Group) ([]*admin.Member, error) {
+	members := make([]*admin.Member, 0)
+
+	groupMembers, err := s.google.GetGroupMembers(group)
+	if err != nil {
+		return nil, err
+	}
+
+	seenUsers := make(map[string]bool)
```

Comment:
Instead of mapping to a `bool`, can we map to the user right here only if not seen before... just do the `s.google.GetUser(m.Id)` call right here?